tests/unit/test_portainer.py

```diff
@@ -154,10 +155,11 @@ async def test_create_stack_fails_when_name_contains_uppercase_chars(
     aiohttp_client_session: ClientSession,
     bearer_code: str,
     portainer_stacks: dict[str, Any],
-    valid_docker_stack: dict[str, Any],
+    valid_docker_stack: ComposeSpecsDict,
+    faker: Faker,
 ):
-    swarm_id = 1
-    stack_name = "myAmazingstackname"
+    swarm_id = f"{faker.pyint(min_value=1)}"
+    stack_name = faker.name() + "A"
```

Comment:
```+A```?

Reply:
yes, this test checks that a stack_name with capital letters will raise an exception